### PR TITLE
Add the `-d` / `--stdout` option. Fixes #164.

### LINKED ISSUE #164: New file to stdout
Is it possible to pipe the new file to stdout? Would be nice so that one could use filter in vim :)

### PR DESCRIPTION
**Pull request by akaihola**
- [x] decide on the design (see [this comment](https://github.com/akaihola/darker/issues/164#issuecomment-881945048))
- [x] only print out a single file
- [x] unit tests
- [x] change log

### CODE REVIEW

File: src/darker/__main__.py [hunk 1]
```diff
@@ -233,7 +249,7 @@ def main(argv: List[str] = None) -> int:
     failures_on_modified_lines = False
 
     revrange = RevisionRange.parse(args.revision)
-    write_modified_files = not args.check and not args.diff
+    write_modified_files = not args.check and not args.diff and not args.stdout
```
Comment (levouh) [inline src/darker/__main__.py:252]: Somewhere around here, should there be a check that `len(paths) == 1` when `args.stdout` is truthy?

File: src/darker/config.py [hunk 2]
```diff
@@ -31,21 +32,33 @@ class OutputMode:
...
             return cls.DIFF
         if args.stdout:
             return cls.CONTENT
         return cls.NOTHING
 
     @staticmethod
-    def validate(diff: bool, stdout: bool) -> None:
-        """Raise an exception if both ``diff`` and ``stdout`` are enabled"""
+    def validate_diff_stdout(diff: bool, stdout: bool) -> None:
+        """Raise an exception if ``diff`` and ``stdout`` options are bothenabled"""
```
Comment (levouh) [inline src/darker/config.py:58]: _nit_: `bothenabled` -> `both enabled`

**Review by levouh — COMMENTED** [review]
Useless nitpick @akaihola, going to pull/test and then I'll report back here. Otherwise LGTM.

File: src/darker/config.py (hunk 2 above)
Comment (akaihola) [reply src/darker/config.py]: Thanks, fixed.

**Comment by akaihola** [comment]
@levouh, I rebased on `master` after merging another branch.

**Pull request merged**